eng/.docsettings.yml

```diff
@@ -165,6 +165,7 @@ known_content_issues:
   - ['sdk/provisioning/Azure.Provisioning.ApplicationInsights/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
   - ['sdk/provisioning/Azure.Provisioning.ContainerService/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
   - ['sdk/provisioning/Azure.Provisioning.Deployment/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
+  - ['sdk/provisioning/Azure.Provisioning.RedisEnterprise/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
```

Comment:
i created an issue to remove this in the future: https://github.com/Azure/azure-sdk-for-net/issues/51412
but for this RP, we just adding this exception to work it around.